plugins/book-data-extension/src/plugin/index.ts

```diff
@@ -1,2 +1,28 @@
-export const dunnystring =
-	"This is a dummy string so that the 'build' script doesn't fail (warn = fail in CI) due to empty bundle. Remove when there's something to build.";
+import { BookFetcherPlugin as BookFetcherPluginType, BookEntry } from "@librocco/db";
+import { postMessage } from "./window-helpers";
+import { listenForBook, listenForExtension } from "./listeners";
+
+export const BookFetcherPlugin = (): BookFetcherPluginType => {
```

Comment:
Maybe name this `createBookDataExtensionPlugin`. It's not a convention to name functions in PascalCase, the suggested name is more indicative and the name clashing (which you've solved with aliasing here) would also be present when consuming the plugin in the app.